src/test/java/com/amazonaws/gurureviewercli/util/ZipUtilsTest.java

```diff
@@ -35,4 +55,28 @@ public void test_packUnpack() throws IOException {
             Assertions.assertTrue(expectedFileNames.isEmpty());
         }
     }
+
+    /*
+      If a aws-codeguru-reviewer.yml file is present, it has to be included in the zip file even if the root folder
+      is not mentioned in the list of source directories.
+     */
+    @Test
+    void test_packUnpackWithConfig() throws IOException {
+        val testDir = Paths.get("test-data/fake-repo-with-config");
+        val zipName = workDir.resolve("test.zip").toString();
+        ZipUtils.pack(Arrays.asList(testDir.resolve("src-dir")), testDir, zipName);
+        try (ZipFile zipFile = new ZipFile(zipName)) {
+            Enumeration<? extends ZipEntry> entries = zipFile.entries();
+            val expectedFileNames =
+                new HashSet<String>(Arrays.asList("src-dir/included-src.txt",
+                                                  "aws-codeguru-reviewer.yml"));
+            while (entries.hasMoreElements()) {
+                ZipEntry entry = entries.nextElement();
+                Assertions.assertFalse(entry.toString().contains("\\"), "Unexpected zip entry " + entry);
```

Comment:
Is this intentionally a Windows-only path separator? Is the other case (`/`) relevant at all? I'm not sure how this plays into the purpose of the test.

Reply:
the other case works as expected. The problem is that the extraction of windows separators creates file names with `\` rather than directories. The other case is the default case and gets tested implicitly.